src/allotropy/allotrope/schema_mappers/adm/liquid_chromatography/benchling/_2023/_09/liquid_chromatography.py

```diff
@@ -0,0 +1,389 @@
+            product_manufacturer=data.metadata.product_manufacturer,
+            device_identifier=data.metadata.device_id,
+            firmware_version=data.metadata.firmware_version,
+        )
+
+    def get_chromatography_col_doc(
+        self, chromatography_column_doc: ChromatographyDoc
+    ) -> ChromatographyColumnDocument:
+        return ChromatographyColumnDocument(
+            chromatography_column_serial_number=chromatography_column_doc.chromatography_serial_num,
+            chromatography_column_chemistry_type=chromatography_column_doc.chromatography_chemistry_type,
+            column_inner_diameter=TQuantityValueMillimeter(
+                value=chromatography_column_doc.column_inner_diameter,
+            ),
+            chromatography_column_particle_size=TQuantityValueMicrometer(
+                value=chromatography_column_doc.chromatography_particle_size,
+            ),
+        )
+
+    def get_sample_doc(self, sample_doc: SampleDoc) -> SampleDocument:
+        return SampleDocument(
+            sample_identifier=sample_doc.sample_identifier,
+            batch_identifier=sample_doc.batch_identifier,
+        )
+
+    def get_injection_doc(self, injection_doc: InjectionDoc) -> InjectionDocument:
+        return InjectionDocument(
+            injection_identifier=injection_doc.injection_identifier,
+            injection_time=self.get_date_time(injection_doc.injection_time),
+            autosampler_injection_volume_setting__chromatography_=TQuantityValueCubicMillimeter(
+                value=injection_doc.autosampler_injection_volume_setting,
+            ),
+        )
+
+    def get_processed_data_agg_doc(
+        self, processed_data_doc: ProcessedDataDoc
+    ) -> ProcessedDataAggregateDocument:
+        return ProcessedDataAggregateDocument(
+            processed_data_document=[
+                ProcessedDataDocumentItem(
+                    chromatogram_data_cube=(
+                        self.get_data_cube(
+                            processed_data_doc.chromatogram_data_cube,
+                            TDatacube,
+                        )
+                        if processed_data_doc.chromatogram_data_cube
+                        else None
+                    ),
+                    derived_column_pressure_data_cube=(
+                        self.get_data_cube(
+                            processed_data_doc.derived_column_pressure_data_cube,
+                            DerivedColumnPressureDataCube,
+                        )
+                        if processed_data_doc.derived_column_pressure_data_cube
+                        else None
+                    ),
+                )
+            ]
+        )
+
+    def get_device_control_doc_item(
+        self, device_control_doc: DeviceControlDoc
+    ) -> DeviceControlDocumentItem:
+        return DeviceControlDocumentItem(
+            device_type=device_control_doc.device_type,
+            solvent_concentration_data_cube=(
```

Comment:
The `if None` pattern seems like it would benefit from moving into the function. 

Make `get_data_cube` data cube argument optionally be None, and return None in that case, then these all simplify to just calling `get_data_cube`